A14-channelz.md

```diff
@@ -741,31 +741,38 @@ message ChannelData {
 message ChannelTraceEvent {
   // High level description of the event.
   string description = 1;
-  // Status/error associated with this event. 
+  // Status/error associated with this event.
+  // Optional, only present is Status != OK. 
   google.rpc.Status status = 2;
   // When this event occurred.
   google.protobuf.Timestamp event_timestamp = 3;
   // The connectivity state the channel was in when this event occurred.
   ChannelConnectivityState state = 4;
-  // uuid of referenced subchannel.
+  // channel_id of referenced subchannel.
```

Comment:
as mentioned, this should probably be a ref.   Also, this could be a channel or subchannel.

Reply:
Done